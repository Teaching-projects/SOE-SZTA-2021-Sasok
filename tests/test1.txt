150
15
1
10
15
2
30
170
13
1