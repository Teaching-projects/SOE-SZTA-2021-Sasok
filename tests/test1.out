Első egység HP-ja: Első egység DMG-je: Második egység HP-ja: Második egység DMG-je: 
A Csata elkezdődött! Warrior kezdheti a támadást Shaman ellen.
1. körben Warrior támad --- Warrior élete: 150 || Shaman élete: 155
2. körben Shaman támad, ---Warrior élete: 137 || Shaman élete: 155
2. körben Warrior támad --- Warrior élete: 137 || Shaman élete: 140
3. körben Shaman támad, ---Warrior élete: 124 || Shaman élete: 140
3. körben Warrior támad --- Warrior élete: 124 || Shaman élete: 125
4. körben Shaman támad, ---Warrior élete: 111 || Shaman élete: 125
4. körben Warrior támad --- Warrior élete: 111 || Shaman élete: 110
5. körben Shaman támad, ---Warrior élete: 98 || Shaman élete: 110
5. körben Warrior támad --- Warrior élete: 98 || Shaman élete: 95
6. körben Shaman támad, ---Warrior élete: 85 || Shaman élete: 95
6. körben Warrior támad --- Warrior élete: 85 || Shaman élete: 80
7. körben Shaman támad, ---Warrior élete: 72 || Shaman élete: 80
7. körben Warrior támad --- Warrior élete: 72 || Shaman élete: 65
8. körben Shaman támad, ---Warrior élete: 59 || Shaman élete: 65
8. körben Warrior támad --- Warrior élete: 59 || Shaman élete: 50
9. körben Shaman támad, ---Warrior élete: 46 || Shaman élete: 50
9. körben Warrior támad --- Warrior élete: 46 || Shaman élete: 35
10. körben Shaman támad, ---Warrior élete: 33 || Shaman élete: 35
10. körben Warrior támad --- Warrior élete: 33 || Shaman élete: 20
11. körben Shaman támad, ---Warrior élete: 20 || Shaman élete: 20
11. körben Warrior támad --- Warrior élete: 20 || Shaman élete: 5
12. körben Shaman támad, ---Warrior élete: 7 || Shaman élete: 5
12. körben Warrior támad --- Warrior élete: 7 || Shaman élete: -10

WARRIOR GYOZEDELMESKEDETT!
